src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -66,12 +61,16 @@
     private static final Class<FeatureInput> FEATURE_ARGUMENT_CLASS = FeatureInput.class;
 
     /**
-     * At startup, walk through the packages in CODEC_PACKAGES, and save any (concrete) FeatureCodecs discovered
+     * At startup, walk through the packages in codec packages, and save any (concrete) FeatureCodecs discovered
      * in DISCOVERED_CODECS
      */
     static {
+
+        // Get our configuration:
+        final GATKConfig config = ConfigCache.getOrCreate( GATKConfig.class );
+
         final ClassFinder finder = new ClassFinder();
-        for ( final String codecPackage : CODEC_PACKAGES ) {
+        for ( final String codecPackage : config.codec_packages() ) {
```

Comment:
It would also requires a `ignore_codec_classes` to allow removal of single codecs, in case of clash (two codecs that can decode the same file will throw).

Maybe, for symmetry, it will also require `codec_classes`. But I do not need that feature yet...